proto/connector/v1/connector.proto

```diff
@@ -5,16 +5,14 @@ package connector.v1;
 import "google/protobuf/descriptor.proto";
 import "opencdc/v1/opencdc.proto";
 
-option (metadata_conduit_version) = "conduit.version";
```

Comment:
Is this being removed, since records are now defined by OpenCDC (and not by Conduit), so the Conduit version isn't relevant anymore?

Reply:
Oh I should have put this change into the previous PR. I removed it since the Conduit version shouldn't be relevant for the connector, also it's a thing that won't change in the lifetime of a connector so putting it onto the record as metadata makes less sense. If the connector would really be interested in this we should rather pass it in the config options in `Configure` or attach it to the context.

Reply:
I moved the change to https://github.com/ConduitIO/conduit-connector-protocol/pull/8